2-mandatory/4-stocks.js

```diff
@@ -65,6 +86,27 @@ function getPriceChanges(closingPricesForAllStocks) {
 */
 function highestPriceDescriptions(closingPricesForAllStocks, stocks) {
     // TODO
+    let theHighestPrice = highestPrice(closingPricesForAllStocks);
+    console.log(theHighestPrice)
+    let report = [];
+    let i = 0;
+    for (let stock of stocks) {
+        report.push("The highest price of " + stock.toUpperCase() + " in the last 5 days was " + theHighestPrice[i])
+        i++;
+    }
+    return report;
+
+}
+
+function highestPrice(closingPrices) {
+    highestPrices = [];
+    for (let price of closingPrices) {
+        highestPrices.push(/*Number*/((Math.max.apply(null, price)).toFixed(2))) // Found this solution here: https://stackoverflow.com/questions/1669190/find-the-min-max-element-of-an-array-in-javascript
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/max Please write about finding max number in array. I found the easiest way for me to find max number in array using spread syntax.
const arr = [1, 2, 3];
const max = Math.max(...arr);